README.md

```diff
@@ -119,72 +151,95 @@ calculations. Before the TLS connection is complete, a shared secret key
 must be negotiated between the server and the client. This key is used
 to encrypt all future communications during the connection.
 
-### **MQTT Connection** 
+### MQTT Connection
 
 After successfully connecting on the TLS level, the board starts
 establishing the MQTT connection. Since the TLS handles authentication
 and security, MQTT does not have to provide a username or password.
 
 <img src=".//media/image8.ppm"/>
 
-Checklist
-=========
+## Prerequisites
+
+- [Setup your Azure IoT hub](https://github.com/Azure/azure-iot-device-ecosystem/blob/master/setup_iothub.md)
+- [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/releases)
+- For the Plug and Play sample, download the Plug and Play DTMI [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json)
+- [MPLAB X IDE V5.30 or later](https://www.microchip.com/mplab/mplab-x-ide)
+- [XC16 Compiler v1.50 or later](https://www.microchip.com/mplab/compilers)
+
+## Sample Descriptions
+
+### AzureIotDps.X
+
+Connect the device through Azure IoT Hub Device Provisioning to then connect to your IoT Hub.
+
+### AzureIotPnpDps.X
+
```

Comment:
I think we should add this paragraph to clarify: 

"This is the secondary sample that leverages the primary sample work to demonstrate how the PIC-IoT board can be benefits from new Azure offering called IoT Plug and Play. As the result, in addition to view the device telemetry, you can all the capabilities (such as its properties, telemetry, and commands) of the device as well. In short, the key enabled Azure offerings are IoT Hub, DPS, Azure IoT Explorer and IoT plug and Play.

To further understand what Plug and Play is, please see documentation [here](https://docs.microsoft.com/en-us/azure/iot-pnp/overview-iot-plug-and-play).

This sample uses the example interface [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). "